### DISCUSSION TITLE
征集：版权与引用格式

### DISCUSSION THREAD

**Discussion by CLCK0622** [FRC-China/FRC-Wiki#26] [Q&A]
如题，引用格式其实不用特别严谨，简明为主，需要反映对应license的特点。
并征集本项目应当使用的license。
（其实gpl3是copyleft，页脚写错了，但是是创立时临时定的这个license，还不确定）

**Comment by EvanLuo42** [comment]
gpl3 没有禁止 commercial 吧？感觉还是得挑个 non-commercial 的。

**Reply by CLCK0622** [reply]
对的 这就是我想要换license的原因

**Comment by CLCK0622** [comment]
https://creativecommons.org/licenses/by-sa/4.0/deed.zh

**Comment by CLCK0622** [comment]
https://github.com/FRC-China/FRC-Wiki/blob/change-license/docs/start/copyright.md

**Comment by CLCK0622** [comment]
done in #27
